Gruntfile.js

```diff
@@ -474,6 +474,25 @@ module.exports = function(grunt) {
                 reloadOnRestart: true,
                 watchTask: true
             }
+        },
+
+        validation: {
+            options: {
+                reset: grunt.option('reset') || false,
+                stoponerror: false,
+                maxTry: 3,
+                relaxerror: ['Bad value X-UA-Compatible for attribute http-equiv on element meta.'], //ignores these errors 
+                generateReport: true,
+                errorHTMLRootDir: "w3cErrorFolder",
```

Comment:
Wherever the output is placed, we'll need to add it to the `.gitignore` file so it's not committed to the repository